### PR TITLE
Remove Yammer

### PR DESCRIPTION
**Pull request by fletcherdodd**

### CODE REVIEW

**Review by Carlgo11 — COMMENTED** [review]
I'm not sure if listing Viva Engage is the best thing. As it's a part of the Microsoft 365 suite it might be better to remove Yammer completely as Microsoft 365 is already listed here.

**Comment by fletcherdodd** [comment]
> I'm not sure if listing Viva Engage is the best thing. As it's a part of the Microsoft 365 suite it might be better to remove Yammer completely as Microsoft 365 is already listed here.

I definitely agree, I thought about doing this but opted to updated it instead. Will remove it shortly.

**Comment by hkamran80** [comment]
Should `yammer.com` be added to the Microsoft 365 entry as an additional domain?

**Comment by fletcherdodd** [comment]
> Should `yammer.com` be added to the Microsoft 365 entry as an additional domain?

Good thinking! Done.

File: entries/o/office.com.json [hunk 1]
```diff
@@ -5,7 +5,9 @@
       "login.microsoftonline.com",
       "office.microsoft.com",
       "microsoft365.com",
-      "office365.com"
+      "office365.com",
+      "cloud.micrsoft",
```
Comment (hkamran80) [inline entries/o/office.com.json]:
```suggestion
      "cloud.microsoft",
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**